packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -677,43 +735,29 @@ export class IntervalCollectionView<TInterval extends ISerializableInterval> ext
         intervalType: MergeTree.IntervalType,
         props?: MergeTree.PropertySet,
     ) {
-        let seq = 0;
-        if (this.client) {
-            seq = this.client.getCurrentSeq();
-        }
+        const interval: TInterval = this.localCollection.addInterval(start, end, intervalType, props);
 
-        const serializedInterval: ISerializedInterval = {
-            end,
-            intervalType,
-            properties: props,
-            sequenceNumber: seq,
-            start,
-        };
-
-        return this.addInternal(serializedInterval, true, undefined);
-    }
-
-    public delete(
-        start: number,
-        end: number) {
-        let sequenceNumber = 0;
-        if (this.client) {
-            sequenceNumber = this.client.getCurrentSeq();
+        if (interval) {
+            const serializedInterval = {
+                end,
+                intervalType,
+                properties: interval.properties,
+                sequenceNumber: this.client?.getCurrentSeq() ?? 0,
```

Comment:
>sequenceNumber [](http://example.com/codeflow?start=16&length=14)

we would usually call this referenceSequenceNumber, as sequenceNumbers are provided by the server

Reply:
looks to be pre-existing

Reply:
Do you think the member name in ISerializedInterval should be changed?